compass/landice/tests/thwaites/restart_test/__init__.py

```diff
@@ -0,0 +1,78 @@
+from compass.validate import compare_variables
+from compass.testcase import TestCase
+from compass.landice.tests.thwaites.run_model import RunModel
+
+
+class RestartTest(TestCase):
+    """
+    A test case for performing two MALI runs of the Thwaites setup,
+    one full run and one run broken into two segments with a restart.  The
+    test case verifies that the results of the two runs are identical.
+    """
+
+    def __init__(self, test_group):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.landice.tests.thwaites
+            The test group that this test case belongs to
+
+        """
+        name = 'restart_test'
+        subdir = '{}'.format(name)
+        super().__init__(test_group=test_group, name=name, subdir=subdir)
+
+        cores = 36
+        min_cores = 4
+
+        name = 'full_run'
+        step = RunModel(test_case=self, name=name, subdir=name, cores=cores,
+                        min_cores=min_cores, threads=1)
+        # modify the namelist options and streams file
+        step.add_namelist_file(
+            'compass.landice.tests.greenland.restart_test',
+            'namelist.full', out_name='namelist.landice')
+        step.add_streams_file(
+            'compass.landice.tests.greenland.restart_test',
+            'streams.full', out_name='streams.landice')
```

Comment:
```suggestion
        step.add_namelist_file(
            'compass.landice.tests.thwaites.restart_test',
            'namelist.full', out_name='namelist.landice')
        step.add_streams_file(
            'compass.landice.tests.thwaites.restart_test',
            'streams.full', out_name='streams.landice')
```

Reply:
If this is on purpose, it would be better to have these files in some shared (framework) space so it's clear that they're shared between test cases.  But I suspect it's just a typo.